compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -17388,82 +17388,47 @@ TR::Node *lucmpSimplifier(TR::Node *node, TR::Block *block, TR::Simplifier * s)
 
 TR::Node *imaxminSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    {
+   TR_ASSERT(node->getNumChildren() == 2, "max/min node should only support 2 children");
```

Comment:
We've been trying to move away from non fatal asserts as they are only enabled in debug builds which we do not have automation for and people more seldom build them. I would propose we remove these asserts (same with ones below) since they are not active by default and the verifier will have already caught issues of wrong number of children or incorrect child types.

Reply:
Fixed in a2fb988